src/ContactPage/Contact.scss

```diff
@@ -2,45 +2,51 @@
 @import "../styles/layout";
 
 .contact-section {
+  background-image: url(../../assets/letters.jpg);
+  
+  
+  height:calc(100vh - 60px);
+  background-size: cover;
   display: flex;
   flex-direction: column;
   flex-wrap: wrap;
 
   &__border {
     padding: 2% 0;
-    background: rgba(42, 71, 110, 0.918);
     width: 100%;
+    background-color: grey;
+    opacity: 0.9;
+    margin-top: 8%;
```

Comment:
This is more of a design comment but does this need a margin-top? Feels a bit unneccessary